lms/static/scripts/frontend_apps/api-types.ts

```diff
@@ -196,7 +201,7 @@ export type AutoGradingGrade = {
   /** Grade that was last synced, if any */
   last_grade: number | null;
   /** When did the last grade sync happen, if any */
-  last_grade_date: number | null;
+  last_grade_date: ISODate | null;
```

Comment:
This was a typo. The type was never `number`, but we are currently not using this anywhere.